tursu/tursu/urls.py

```diff
@@ -19,4 +19,7 @@
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('user/', include('registered_user.urls')),
+    path('search/', include('search.urls')),
+    path('category', include('category.urls')),
```

Comment:
Slash is forgotten at the end of category?

Reply:
Yes, I just noticed I was sending the HTTP requests like "http://127.0.0.1:8000/category?name=Electronics" which is wrong. I just fixed it.